fcrepo-kernel-modeshape/src/test/resources/config/testing/repository.json

```diff
@@ -20,8 +20,8 @@
         "providers" : [
             {
                 "name" : "Servlet Provider",
-                "classname" : "org.modeshape.jcr.security.ServletProvider",
-            },
+                "classname" : "org.modeshape.jcr.security.ServletProvider"
+            }
```

Comment:
Modeshape must use a pretty lenient JSON parser for this to have been passing before, interesting, but the file does appear to be getting used.